localnet/README.md

```diff
@@ -15,12 +15,20 @@ $(go env GOPATH)/src/github.com/harmony-one/harmony-test
 * The localnet faucet address is `one1zksj3evekayy90xt4psrz8h6j2v3hla4qwz4ur` 
 
 ## Requirements (dev testing - no docker)
+* Go
+* jq
```

Comment:
why is `jq` a requirement?

Reply:
That's because of the `make test` which runs the ./scripts/run.sh target for dev testing as it is stated in the README, running without docker.
https://github.com/harmony-one/harmony-test/blob/master/localnet/scripts/run.sh#L132